syftlib/src/main/java/org/openmined/syft/proto/SyftModel.kt

```diff
@@ -28,13 +35,27 @@ data class SyftModel(
 //        } ?: return null
 //    }
 
+    /**
+     * This method is used to save/update SyftModel class.
+     * call this function after every gradient step.
```

Comment:
Caps in call. better suited would be 
- This function must be called after every gradient step to update the model state for further plan executions